### ISSUE TITLE
inconsistency in web apps when selecting an item

### ISSUE THREAD

**Issue by TimSnaith** [OnePointGlobal/OnePoint-Global-Account-Toolkit#181]
I was looking at the web apps and i really like the coloured tick boxes of the surveys app 
![image](https://user-images.githubusercontent.com/26327747/27373362-e7d59612-565f-11e7-9d0b-802db229d165.png)

but all the others are very plain and boring tick boxes, is there a reason? or can we hav ethe style for surveys in all areas please

![image](https://user-images.githubusercontent.com/26327747/27373395-00b1ca2a-5660-11e7-8ae1-7b385b1d6785.png)

**Comment by TimSnaith** [comment]
also the alignment is different across this surveys app and all the others. see below 
![image](https://user-images.githubusercontent.com/26327747/27374018-3d7d25b0-5662-11e7-97f8-0985fdd88e79.png)

and now see the page is in line with the upper guide line 

![image](https://user-images.githubusercontent.com/26327747/27374072-6c18654c-5662-11e7-8fdd-91444db1a3ac.png)

**Comment by meghanavbharadwaj** [comment]
For survey module- align in line with the upper guide line.
Check boxes - As per our discussion in the meeting, it was decided that we will remove those yellow tick icons and inside each survey we will indicate for each module whether it is completed or not. So that the user gets a clear picture of the steps that are remaining. We had decided to take it up as enhancement.
We can give an attractive icon for checkbox.

**Comment by TimSnaith** [comment]
I think we should follow what google do as were using their material design so as below for google mail you can see the use of icons and circles like we have in Surveys
![image](https://user-images.githubusercontent.com/26327747/27404817-568eac44-56c7-11e7-94e3-71c1ffaf2aa8.png)

**Comment by veereshms** [comment]
As per new design applied all points mentioned above are fixed except below, @meghanavbharadwaj : Is it according to design?, please provide your inputs.

but all the others are very plain and boring tick boxes, is there a reason? or can we hav ethe style for surveys in all areas please

![image](https://user-images.githubusercontent.com/26476914/28362187-1ff0db7a-6c99-11e7-8d3f-b82a85ec7790.png)

**Comment by meghanavbharadwaj** [comment]
As per Tim's feedback, we have changed the design for listing and multiple delete according to material design. In Participants and Themes module, there are two listings. So, to differentiate between outer listing and inner listing we have changed in outer listing by keeping a normal checkbox and a highlight for it. 
![outer_listing](https://user-images.githubusercontent.com/28921540/28363646-ebffc1a4-6c9e-11e7-9a09-4f9caf682cc1.jpg)
